src/daidepp/grammar.py

```diff
@@ -77,10 +80,19 @@ def from_level(level: DAIDE_LEVEL):
     "wve": 'power ws "WVE"',
     "unit_type": '"AMY" / "FLT"',
     "unit": "power ws unit_type ws province",
-    "province": "prov_coast / prov_no_coast / prov_sea / (lpar prov_coast ws coast rpar)",
+    "prov_no_coast": "prov_land_sea / prov_landlock / prov_sea",
```

Comment:
Per the DAIDE spec, prov_no_coast "is a province with no coast specification." So it's just any province with or without a coast specification, not just provinces that have no coast.